autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/viewing/AutofillManagementListMode.kt

```diff
@@ -242,16 +242,15 @@ class AutofillManagementListMode : DuckDuckGoFragment(R.layout.fragment_autofill
         binding.logins.gone()
     }
 
-    private fun renderCredentialList(credentials: List<LoginCredentials>) {
+    private suspend fun renderCredentialList(credentials: List<LoginCredentials>) {
         binding.emptyStateLayout.emptyStateContainer.gone()
         binding.logins.show()
 
         val currentUrl = getCurrentUrlForSuggestions()
-        val suggestions = suggestionMatcher.getSuggestions(currentUrl, credentials)
+        val directSuggestions = suggestionMatcher.getDirectSuggestions(currentUrl, credentials)
```

Comment:
shouldn't be all this suggestions already in the `ViewState` instead of executing this logic from the UI?

Not a blocker since the previous version was already executing similar logic in here, but to consider moving this logic out of the UI

Reply:
Good suggestion. I'm going to capture this in a separate task for the backlog: [Autofill improvement: Move more logic out of fragment and into ViewModel](https://app.asana.com/0/1203115518054842/1205221825961800/f)